src/com/hms/flexyosisoftconnector/DataPoster.java

```diff
@@ -7,14 +7,40 @@
    private static final int maxDataPointsPerPost = 200;
    private boolean shouldRun;
 
-   public void run()
+   public DataPoster (int communicationType) {
+	   switch(communicationType) {
+       case OSIsoftConfig.omf:
+    	   Logger.LOG_INFO("OMF support enabled");    	   
+           break;
+       case OSIsoftConfig.piwebapi:
+    	   Logger.LOG_INFO("Using connection settings for piwebapi 2018 and earlier");
+           break;
+       default:
+           Logger.LOG_ERR("Immpossible to reach here, checks done in config class.");
+           break;
+       }
+   }
+
+   public void run(int communicationType)
    {
       shouldRun = true;
 
       while(shouldRun)
       {
 
-         OSIsoftServer.startBatch();
+          switch(communicationType) {

```

Comment:
Indentation for Dataposter is wrong for all new added lines.  Looks like you are using 4 spaces instead of 3

Reply:
i think some files use 3 and some use 4. this could be changed in a separate commit to align all the files one one convention.